pkg/registry/pod/etcd/etcd.go

```diff
@@ -39,6 +39,9 @@ import (
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/util/fielderrors"
 )
 
+// Clear terminated pods from etcd after 4 days.
+var ttlForTerminatedPods uint64 = 4 * 24 * 3600
```

Comment:
Can we make this configurable?  

There are some pods that are succeed/failed that we may want to keep longer than others.

@smarterclayton - thoughts?